src/app/zap-templates/zcl/air-pressure-measurement.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<configurator>
+  <domain name="CHIP"/>
+  <cluster>
+    <domain>Measurement &amp; Sensing</domain>
+    <name>Air Pressure Measurement</name>
+    <code>0x0407</code>
+    <define>AIR_PRESSURE_MEASUREMENT_CLUSTER</define>
+    <client init="false" tick="false">true</client>
+    <server init="false" tick="false">true</server>
+    <description>Air pressure measurement cluster in hPA with 1 digit fixed point resolution</description>
+    <attribute side="server" code="0x0000" define="AIR_PRESSURE_MEASUREMENT_MEASURED_VALUE" type="INT16U" min="8700" max="11000" writable="false" default="10132" optional="false">measured value</attribute>
+    <attribute side="server" code="0x0001" define="AIR_PRESSURE_MEASUREMENT_ALTITUDE" type="INT16S" min="-32768" max="32767" writable="true" default="0" optional="false">altitude</attribute>
```

Comment:
Is `writable=true` expected ?

Reply:
Yes altitude is writable. It should be set by the "user", to indicate the hight above / below see level.

Reply:
Well, in our scenarios the device doesn't know that. Because it has no GPS. The altitude is set by the an app during the setup / paring process of the accessory. The app on the mobile phone know the current location & altitude and propagates it to the accessory.

Reply:
The problem is that the device cannot measure the altitude by itself. And the altitude measurement available on some devices is indeed using an air pressure sensor which derives the altitude from the air pressure value and data from the internet.
So the controller has to set the altitude to get correct air pressure readings. Typically this value can be auto-generated on the controller by getting it from some internet services based on the geo-location. If this is not possible then the User has to specify the altitude value.

Reply:
Ok, thanks for clarifying :)